src/interaction/sessionrecording_lua.inl

```diff
@@ -98,15 +112,15 @@ int startPlayback(lua_State* L, interaction::KeyframeTimeRef timeMode,
     global::sessionRecording->startPlayback(
         const_cast<std::string&>(playbackFilePath),
         timeMode,
-        forceSimTimeAtStart
+        forceSimTimeAtStart,
+        loop
     );
 
     ghoul_assert(lua_gettop(L) == 0, "Incorrect number of items left on stack");
     return 0;
 }
 
 int startPlaybackDefault(lua_State* L) {
-    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::startPlaybackDefault");
```

Comment:
Instead of removing this check (which is really useful as it provides meaningful error info to the user), you can specify that one to two arguments are allowed like this: 
```
ghoul::lua::checkArgumentsAndThrow(L, {1, 2}, "lua::startPlaybackDefault");
```

Reply:
Done